terraform/bastion/Jenkinsfile-check

```diff
@@ -0,0 +1,54 @@
+library("tdr-jenkinslib")
+def repo = "tdr-scripts"
+
+pipeline {
+  agent {
+    label "master"
+  }
+  parameters {
+    choice(name: "STAGE", choices: ["intg", "staging", "prod"], description: "The stage you are checking the age of")
+  }
+  stages {
+    stage("Run git secrets") {
+      steps {
+        script {
+          tdr.runGitSecrets(repo)
+        }
+      }
+    }
+    stage('Destroy bastion if the instance is older than 7 days') {
```

Comment:
It'll be interesting to see how aggressive this seems in practice. 🙂 One of the nice things about Cloud Custodian is the ability to mark things for deletion, so that you can prevent it if necessary. But we really don't want to keep bastions around for too long, so let's go with this for now!

Reply:
Yeah ok. Hopefully once the API does reporting we'll hardly need them at all.